### ISSUE TITLE
Pickling broken in non-embedded contexts

### ISSUE THREAD

**Issue by skef** [fontforge/fontforge#3518]
All of the below applies to python 3. The code works in python2. 

Running `python tests/test932.py` on the (updated) #3513 code results in:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named '__FontForge_Internals___'
Segmentation fault (core dumped)
```
This is because `PyFFPoint_pickleReducer()` tries to load the module `__FontForge_Internals___` if `_new_point` (a static) is NULL. Unfortunately in python 3 `__FontForge_Internals___` is not enough of a real module entity outside of the embedded context to load. 

This is all ultimately a side-effect of the partial fix in #2891. That fix works in general (otherwise `psMat` would still be broken), but it relies on each python 3 module having its own python-level identity. So if that is going to remain the way we do module plumbing, I'm pretty sure we need to add the files for `__FontForge_Internals___` to `pyhook`. 

@jtanx what do you think?

**Comment by jtanx** [comment]
Hmm, it might have been that the pickling was only meant as an internal thing when saving sfds, although I'm not completely sure.

It seems a little bit gross to have to expose that internal module to get it to work; I'll take a look later to see if there's some alternative

**Comment by skef** [comment]
I have only looked at this for a very short time (your fault!). I wasn't able to determine why it's set up the way it is. But one possibility that would make sense is that you would *expect* to be able to unpickle even if  you don't have some module that defines an object imported. Or at least, that seems right (doesn't it?).

If that *is* right, then there needs to be some means for the pickler to import the appropriate modules for itself as its unpacking. And if *that* is right then you can't rely on `fontforge` itself being imported.

**Comment by jtanx** [comment]
Yeah, I think it makes sense to be able to use that pickling feature from pyhook, given that you can load sfds from that too. Anyway, modifying `sys.modules` works, both for python 2 and 3. I'm pretty sure we weren't doing the right thing even on python 2, it just so happened that it worked...

**Comment by jtanx** [comment]
Keeping open for discussion on what to do with that internal module
